packages/backend-ai/API.md

```diff
@@ -1,74 +1,74 @@
-    tools?: Array<ExecutableTool<runtime.JSONSchema, any>>;
-}) => Promise<void>;
-
-// @public (undocumented)
-type ToolResultContentBlock = runtime.ToolResultContentBlock;
-
-// (No @packageDocumentation comment for this package)
-
-```
+## API Report File for "@aws-amplify/backend-ai"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { ConstructFactory } from '@aws-amplify/plugin-types';
+import { ConversationTurnEventVersion } from '@aws-amplify/ai-constructs/conversation';
+import { FunctionResources } from '@aws-amplify/plugin-types';
+import { ResourceProvider } from '@aws-amplify/plugin-types';
+import * as runtime from '@aws-amplify/ai-constructs/conversation/runtime';
+
+declare namespace __export__conversation {
+    export {
+        ConversationHandlerFunctionFactory,
+        DefineConversationHandlerFunctionProps,
+        defineConversationHandlerFunction
+    }
+}
+export { __export__conversation }
+
+declare namespace __export__conversation__runtime {
+    export {
+        ToolResultContentBlock,
+        ExecutableTool,
+        ConversationTurnEvent,
+        handleConversationTurnEvent,
+        createExecutableTool
+    }
+}
+export { __export__conversation__runtime }
+
+// @public (undocumented)
+type ConversationHandlerFunctionFactory = ConstructFactory<ResourceProvider<FunctionResources>> & {
+    readonly eventVersion: ConversationTurnEventVersion;
+};
+
+// @public (undocumented)
+type ConversationTurnEvent = runtime.ConversationTurnEvent;
+
+// @public (undocumented)
+const createExecutableTool: <TJSONSchema extends runtime.JSONSchema = runtime.JSONSchema, TToolInput = runtime.FromJSONSchema<TJSONSchema>>(name: string, description: string, inputSchema: runtime.ToolInputSchema<TJSONSchema>, handler: (input: TToolInput) => Promise<ToolResultContentBlock>) => ExecutableTool<TJSONSchema, TToolInput>;
+
+// @public
+const defineConversationHandlerFunction: (props: DefineConversationHandlerFunctionProps) => ConversationHandlerFunctionFactory;
+
+// @public (undocumented)
+type DefineConversationHandlerFunctionProps = {
+    name: string;
+    entry?: string;
+    models: Array<{
+        modelId: string | {
+            resourcePath: string;
+        };
+        region?: string;
+    }>;
+};
+
+// @public (undocumented)
+type ExecutableTool<TJSONSchema extends runtime.JSONSchema = runtime.JSONSchema, TToolInput = runtime.FromJSONSchema<TJSONSchema>> = runtime.ToolDefinition<TJSONSchema> & {
+    execute: (input: TToolInput) => Promise<ToolResultContentBlock>;
+};
+
+// @public (undocumented)
+const handleConversationTurnEvent: (event: ConversationTurnEvent, props?: {
+    tools?: Array<ExecutableTool<runtime.JSONSchema, any>>;
+}) => Promise<void>;
+
+// @public (undocumented)
+type ToolResultContentBlock = runtime.ToolResultContentBlock;
+
+// (No @packageDocumentation comment for this package)
+
+```
```

Comment:
im not sure why this file is here, perhaps you have to pull from main again

Reply:
yep, this file shouldn't be here.
and it seems it has whitespace changes.

for the purpose of this pr just remove this from diff please.

to prevent this from happening you can try the following:
1. What @awsluja said above - merge latest main
2. Clean - rebuild
3. Check these settings https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings (the diff seems to be whitespace changes).